apps/src/applab/DesignModeHeaders.jsx

```diff
@@ -84,23 +85,26 @@ export default class DesignModeHeaders extends React.Component {
         display: this.props.isToolboxVisible ? 'flex' : 'none',
         justifyContent: 'space-between',
         width: 270,
-        borderRight: '1px solid gray',
-        float: 'left',
+        borderLeft: this.props.isRtl ? '1px solid gray' : '',
```

Comment:
Non-blocking but if you have time, these could be put in a separate css module - kinda busy in this react file.

Reply:
Yeeeeaaahhhh, I decided against it cause it would take more time than it's worth rn 😶‍🌫️